content/fr/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -0,0 +1,303 @@
+---
+title: kubeadm init
+content_template: templates/concept
+weight: 20
+---
+{{% capture overview %}}
+Cette commande initialise un noeud Kubernetes control-plane.
+{{% /capture %}}
+
+{{% capture body %}}
+
+{{< include "generated/kubeadm_init.md" >}}
+
+### Séquence d'initialisation {#init-workflow}
+`kubeadm init` assemble un noeud Kubernetes control-plane en effectuant les étapes suivantes :
+
+1. Exécute une série de contrôles pour valider l'état du système avant d'y apporter des changements.
+   Certaines validations peuvent émettre seulement des avertissements (warnings), 
+   d'autres peuvent générer des erreurs qui forceront l'interruption de kubeadm 
+   jusqu'à ce que le problème soit résolu
+   ou jusqu'à ce que l'utilisateur spécifie `--ignore-preflight-errors=<list-des-erreurs>`.
+
+1. Génère une autorité de certification (CA) auto signée (ou utilise une existante si spécifiée) pour
+   installer les identités de chaque composant du cluster. Si l'utilisateur a fourni son propre certificat 
+   et/ou clef de CA en le (la) copiant dans le répertoire des certificats, configuré avec  `--cert-dir`
+   (`/etc/kubernetes/pki` par défaut) cette étape est sautée comme expliqué dans le document
+   [utiliser ses propres certificats](#custom-certificates).
+   Les certificats de l'API Server auront des entrées SAN additionnelles pour chaque argument `--apiserver-cert-extra-sans`.
+
+1. Ecrit les fichiers kubeconfig dans `/etc/kubernetes/` pour la
+   kubelet, le controller-manager et l'ordonnanceur (scheduler) 
+   qui seront utlisés pour les connexions à l'API server, chacun avec sa propre identité, 
+   ainsi qu'un fichier kubeconfig supplémentaire pour l'administration, nommé `admin.conf`.
+
+1. Génère des manifestes statiques de Pod pour l'API server,
+   le controller manager et l'ordonnanceur. Au cas où aucun etcd externe n'est fourni, 
+   un manifeste statique de Pod pour etcd est généré.
+
+   Les manifestes statiques de Pod sont écrits dans `/etc/kubernetes/manifestes`; 
+   la kubelet surveille ce répertoire afin que les Pods soient créés au démarrage.
+
+   Dès lors que les pods de control-plane sont démarrés, la séquence de `kubeadm init` peut alors continuer.
+
+1. Applique les étiquettes (labels) et marques (taints) au noeud control-plane afin qu'aucune workload additionnelle ne s'y exécutent.
+
+1. Génère le jeton que les noeuds additionnels peuvent utiliser pour s'enregistrer avec un control-plane. Il est possible que l'utilisateur fournisse un jeton en utilisant `--token`, 
+   comme décrit dans la documentation [à propos du jeton kubeadm](/docs/reference/setup-tools/kubeadm/kubeadm-token/).
```

Comment:
Guess the referenced docs should link to those docs (if already translated) of fr branch?

Reply:
I don't think there's a french translation of this page yet

Reply:
Then that would be ok as-is :)

Reply:
I guess so, yes !